setup.py

```diff
@@ -360,7 +360,7 @@ def generate_cython():
                                    pip.__version__))
             else:
                 raise RuntimeError("Running cythonize failed!")
-        except ImportError:
+        except (ImportError, ModuleNotFoundError):
             raise RuntimeError("Running cythonize failed!")
```

Comment:
If the goal is to improve the exception handling/stacktrace, maybe it's worth using `raise ... from` then
```suggestion
        except (ImportError, ModuleNotFoundError) as exc:
            raise RuntimeError("Running cythonize failed!") from exc
```

Reply:
That seems wrong. Note that the exception is unrelated to the actual build failure, which comes from `subprocess.call`

Reply:
Ah yes, ok I see. Then ignore my comment if we cannot improve the message.